File: app/src/utils/escrow-defaults.ts
// Shared escrow default constants — mirrors backend EscrowDefaults.java
export const ESCROW_DEFAULTS = {
    CURRENCY: "NGN",
    INSPECTION_PERIOD_DAYS: 3,
    AUTO_RELEASE: true,
    DISPUTE_WINDOW_HOURS: 48,
    REQUIRE_PROOF_OF_DELIVERY: true,
    MILESTONE_ENABLED: false,
    AMOUNT_MIN: 100,
    AMOUNT_MAX: 20_000_000,
    FEE_PERCENT: 1.5,
} as const;

File: api/src/main/java/xyz/outlinr/api/utils/EscrowDefaults.java
package xyz.outlinr.api.utils;

public final class EscrowDefaults {

    private EscrowDefaults() {
    }

    public static final String DEFAULT_CURRENCY = "NGN";
    public static final int INSPECTION_PERIOD_DAYS = 3;
    public static final boolean AUTO_RELEASE = true;
    public static final int DISPUTE_WINDOW_HOURS = 48;
    public static final boolean REQUIRE_PROOF_OF_DELIVERY = true;
    public static final boolean MILESTONE_ENABLED = false;
    public static final double AMOUNT_MIN = 100;
    public static final double AMOUNT_MAX = 20_000_000;
    public static final double FEE_PERCENT = 1.5;
}
